x/wasm/keeper/legacy_querier.go

```diff
@@ -122,7 +118,8 @@ func queryContractState(ctx sdk.Context, bech, queryMethod string, data []byte,
 			return nil, sdkerrors.Wrap(err, "json msg")
 		}
 		// this returns raw bytes (must be base64-encoded)
-		return keeper.QuerySmart(ctx, contractAddr, msg)
+		bz, err := keeper.QuerySmart(ctx, contractAddr, msg)
+		return bz, err
```

Comment:
This is changed from `CosmWasm/wasmd-v0.27.0`. We should be the same since we continue to merge from `CosmWasm/wasmd`.
* https://github.com/CosmWasm/wasmd/blob/v0.27.0/x/wasm/keeper/legacy_querier.go#L116